components/Prize/PrizeItem.jsx

```diff
@@ -0,0 +1,18 @@
+require('./PrizeItem.scss')
+
+import React from 'react'
+
+const PrizeItem = ({title,subText}) => {
+  return (
+    <div className="PrizeItem">
+        <div className="PrizeCircle">{title}</div> <span>{subText}</span>
```

Comment:
We should rename `PrizeCircle` to `prize-circle`, since it is not a standalone React component.